colander/tests/test_colander.py

```diff
@@ -1557,6 +1557,13 @@ def test_serialize_string_with_high_unresolveable_high_order_chars(self):
         e = invalid_exc(typ.serialize, node, not_utf8)
         self.assertTrue('cannot be serialized' in e.msg)
 
+    def test_serialize_encoding_with_non_string_type(self):
+        utf8 = '123'.encode('utf-8')
```

Comment:
Hmm, this one should start as text and then encode to bytes (Python2 lets us get away with being sloppy, but that doesn't mean we shouldn't be explicit).